packages/error-reporting/CHANGELOG.md

```diff
@@ -0,0 +1,29 @@
+# Node.js Agent for Google Cloud Errors ChangeLog
```

Comment:
Is this updated manually, or generated? I would just ask to include a `[ci skip]` in the commit title, so we don't have extraneous builds.